comments.php

```diff
@@ -189,12 +189,12 @@ public function onFormProcessed(Event $event)
             case 'addComment':
                 $post = isset($_POST['data']) ? $_POST['data'] : [];
 
-                $lang = filter_var(urldecode($post['lang']), FILTER_SANITIZE_STRING);
-                $path = filter_var(urldecode($post['path']), FILTER_SANITIZE_STRING);
-                $text = filter_var(urldecode($post['text']), FILTER_SANITIZE_STRING);
-                $name = filter_var(urldecode($post['name']), FILTER_SANITIZE_STRING);
-                $email = filter_var(urldecode($post['email']), FILTER_SANITIZE_STRING);
-                $title = filter_var(urldecode($post['title']), FILTER_SANITIZE_STRING);
+                $lang = filter_var(urldecode($_POST['lang']), FILTER_SANITIZE_STRING);
+                $path = filter_var(urldecode($_POST['path']), FILTER_SANITIZE_STRING);
+                $text = filter_var(urldecode($_POST['text']), FILTER_SANITIZE_STRING);
+                $name = filter_var(urldecode($_POST['name']), FILTER_SANITIZE_STRING);
+                $email = filter_var(urldecode($_POST['email']), FILTER_SANITIZE_STRING);
+                $title = filter_var(urldecode($_POST['title']), FILTER_SANITIZE_STRING);
```

Comment:
Why is $post being replaced with $_POST? $post is already a copy of $_POST['data']. Wouldn't you need to add ['data'] to make that work? Even if it did work, why?

Reply:
I verified that this change allows the comments to send an email instead of giving an error.

Reply:
This should be merged immediately. It leads to not a not working comment function...

The version embedded in Lingonberry template should end like this:
```
$lang = filter_var(urldecode($_POST['lang']), FILTER_SANITIZE_STRING);
                $path = filter_var(urldecode($_POST['path']), FILTER_SANITIZE_STRING);
                $text = filter_var(urldecode($_POST['text']), FILTER_SANITIZE_STRING);
                $name = filter_var(urldecode($_POST['name']), FILTER_SANITIZE_STRING);
                $email = filter_var(urldecode($_POST['email']), FILTER_SANITIZE_STRING);
                $title = filter_var(urldecode($_POST['title']), FILTER_SANITIZE_STRING);
```

Otherwise you get the error: `E_NOTICE: undefined index: lang`